core/solver/ir.cpp

```diff
@@ -163,7 +163,8 @@ std::unique_ptr<LinOp> Ir<ValueType>::conj_transpose() const
 template <typename ValueType>
 void Ir<ValueType>::apply_impl(const LinOp* b, LinOp* x) const
 {
-    this->apply_with_initial_guess(b, x, this->get_default_initial_guess());
+    this->apply_with_initial_guess_impl(b, x,
+                                        this->get_default_initial_guess());
```

Comment:
Does this mean it was broken before?

Reply:
if by broken you mean any apply was logged as two nested applies, then yes, it was broken 😆